controller-manager/src/main/resources/templates/PrometheusRules-enmasse.yaml

```diff
@@ -20,6 +20,9 @@ spec:
     - record: enmasse_component_health
       expr: up{job="address-space-controller"} or on(namespace) (1- absent(up{job="address-space-controller"}) )
 
+    - record: enmasse_component_health
+        expr: up{job="job="enmasse-operator-metrics""} or on(namespace) (1- absent(up{job="job="enmasse-operator-metrics""}) )
```

Comment:
Test this out and discovered there's an extra tab at the start of the line which is causing the creation of the resource to fail

Reply:
@robshelly indeed. I updated